sdk/core/azure-core-tracing-opencensus/src/main/java/com/azure/core/tracing/opencensus/OpenCensusTracer.java

```diff
@@ -181,6 +181,12 @@ public Context extractContext(String diagnosticId, Context context) {
         return AmqpPropagationFormatUtil.extractContext(diagnosticId, context);
     }
 
+    @Override
+    public Context getSharedSpanBuilder(String spanName, Context context) {
+        return Context.NONE;
```

Comment:
would it be better to throw an UnsupportedOperationException rather than return something that won't work?